modules/ROOT/pages/constraints/syntax.adoc

```diff
@@ -22,154 +25,89 @@ For constraints that are backed by an index, the index provider for the backing
 Only one valid value exists for the index provider, `range-1.0`, which is the default value.
 There is no supported index configuration for range indexes.
 
-[NOTE]
-====
-Creating a constraint requires the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/database-administration/#access-control-database-administration-constraints[`CREATE CONSTRAINT` privilege].
-====
+[[create-property-uniqueness-constraints]]
+=== Create property uniqueness constraints
 
-[[constraints-syntax-create-node-unique]]
-=== Node property uniqueness constraint
-
-This command creates a property uniqueness constraint on nodes with the specified label and properties.
-
-[source, syntax, role="noheader", indent=0]
+.Syntax for creating a node uniqueness constraint on a single property
+[source, syntax]
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 FOR (n:LabelName)
 REQUIRE n.propertyName IS [NODE] UNIQUE
 [OPTIONS "{" option: value[, ...] "}"]
 ----
 
-[source, syntax, role="noheader", indent=0]
+.Syntax for creating a composite node uniqueness constraint on multiple properties
```

Comment:
composite node property uniqueness constraints were introduced in 4.4, I assume they don't need a label for that

Reply:
No - in the version 5 Manual we only label things introduced in 5.1 and later